README.md

```diff
@@ -63,6 +63,9 @@ on how to run code.
 Read [Install](#install) to generate an installed iTools distribution, a standalone external folder that contains all
 the built objects required to run powsybl programs.
 
+## Build the documentation
+To locally build the readthedocs documentation follow the instructions in the [documentation README](https://github.com/powsybl/powsybl-core/tree/main/docs) page.
```

Comment:
Maybe a direct link to the README.md would be better, since a long tree is displayed before the documentation when using the directory link:

```suggestion
To locally build the readthedocs documentation follow the instructions in the [documentation README](https://github.com/powsybl/powsybl-core/blob/main/docs/README.md) page.
```